test/extended/images/helper.go

```diff
@@ -24,8 +24,8 @@ func GetImageLabels(c client.ImageStreamImageInterface, imageRepoName, imageRef
 }
 
 // ModifySourceCode will modify source code in the pod of the application
-// according to the sed script.
-func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, sedScript, file string) error {
+// according to the provided command.
+func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, cmd []string) error {
```

Comment:
But I do not understand this change in general. You don't end up using a different command than `sed -ie <script> <file>`. The `ModifySourceCode` suddenly doesn't seem as an appropriate name.